data retrieval/src/main/java/de/tinf15b4/kino/retrieval/scraper/KurbelScraper.java

```diff
@@ -45,7 +47,12 @@ private void handleMovies() {
         for (WebElement movieElement : movies) {
             String title = movieElement.findElement(By.xpath(".//h2")).getText();
             Movie movie = saveObject(new Movie(title, null, null, 0, null, null), Movie.class);
-            retrieveMovieInformation(movie);
+            try {
+                movie = retrieveMovieInformation(movie);
+            } catch (MovieDbException e) {
+                getLogger().warn("Failed to retrieve data from movie db for " + title);
+                getLogger().warn(e.toString());
```

Comment:
Make logger in superclass protected and use that one. We dont want to call LoggerFactory (and possibly create a new logger) each time.